doc_src/cmds/fish_status_to_signal.rst

```diff
@@ -0,0 +1,29 @@
+.. _cmd-fish_status_to_signal:
+
+fish_status_to_signal - Convert exit codes to human-friendly signals
+====================================================================
+
+Synopsis
+--------
+
+::
+
+    function fish_prompt
+        echo -n (fish_status_to_signal $pipestatus) (prompt_pwd) '$ '
```

Comment:
This prints:

```fish
SIGINT SIGINT ~ $ 
```

While that *might* be what you want, it might also be surprising, especially since $pipestatus doesn't tend to show as different from $status constantly, so people use this and then it shows up weird sometimes.

Maybe add a `string join` in here?